tideways/provision.sh

```diff
@@ -75,15 +76,31 @@ function install_xhgui_frontend() {
         cd xhgui
         echo " * Installing xhgui"
         sudo php install.php
-        cp -f "${DIR}/config.php" "/srv/www/default/xhgui/config/config.php"
-        echo " * Restarting MongoDB"
-        service mongod restart
     else
         echo -e " * Updating xhgui..."
         cd /srv/www/default/xhgui
         git pull --rebase origin master > /dev/null 2>&1
         noroot composer update --prefer-dist > /dev/null 2>&1
     fi
+    cp -f "${DIR}/config.php" "/srv/www/default/xhgui/config/config.php"
+    if [[ ! -d "/srv/www/default/php-profiler" ]]; then
+        echo -e " * Installing php-profiler for Xhgui"
+        for version in "7.0" "7.1" "7.2" "7.3" "7.4"
+        do
+            if [[ $(command -v php$version) ]]; then
+                apt install "php${version}-sqlite3" -y &>/dev/null
+            fi
+        done
+        cd /srv/www/default
+        mkdir ./php-profiler && cd ./php-profiler
+        echo " * Installing php-profiler"
+        composer require perftools/php-profiler > /dev/null 2>&1
+        composer require perftools/xhgui-collector > /dev/null 2>&1
```

Comment:
should and these and `mkdir` be using `noroot`?

Reply:
composer commands can be aggregated as well:

```
composer require --no-update package1
composer require --no-update package2
composer install (composer update)
```

here I think composer install should be used as previous require commands create .lock file.

Reply:
Thanks for the hint!